Algorithmia/Algorithm/AlgoOption.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AlgoOptions.swift
+//  AlgorithmiaSwiftDemo
+//
+//  Created by Erik Ilyin on 11/8/16.
+//  Copyright © 2016 algorithmia. All rights reserved.
+//
+
+import Foundation
+
+enum AlgoOption {
+    case Timeout(Int)
+    case Stdout(Bool)
+    case RawOutput
+    case VoidOutput
```

Comment:
OK, we have two option for this. 
1. We can build AsyncResponse class and make new method for output=void, like pipeVoid()
2. We can just add new method getRequestId to AlgoResponse
First option will be very user-friendly but user can use option 2